full-stack-tests/presentation/src/backend/PresentationManager.test.ts

```diff
@@ -38,93 +38,73 @@ describe("PresentationManager", () => {
-      const content = await manager.getContent({ imodel, rulesetOrId: ruleset, keys, descriptor: descriptor!, requestContext: new ClientRequestContext(), unitSystem: PresentationUnitSystem.BritishImperial });
-      const displayValues = content!.contentSet[0].displayValues.rc_generic_PhysicalObject_ncc_MyProp_areaElementAspect as DisplayValuesArray;
-      expect(displayValues.length).is.eq(1);
-      expect(((displayValues[0] as DisplayValuesMap).displayValues as DisplayValuesMap)[field.name]!).to.eq("0.1616 ft²");
+      await checkExpectedAreaDisplayValue("0.1616 ft²", PresentationUnitSystem.BritishImperial, imodel, keys, ruleset, defaultFormats);
     });
 
     it("formats property using provided format when it has provided format and default format for requested unit system", async () => {
       const formatProps = {
-        composite:
-        {includeZero:true,
+        composite: {
+          includeZero:true,
           spacer:" ",
-          units:[{label:"ft²",name:"SQ_FT"}],
+          units: [
+            { label:"ft²", name:"SQ_FT" },
+          ],
         },
         formatTraits:"KeepSingleZero|KeepDecimalPoint|ShowUnitLabel",
         precision:4,
         type:"Decimal",
-        uomSeparator:""};
+        uomSeparator:"",
+      };
 
-      const map = {
+      const defaultFormats = {
         // eslint-disable-next-line @typescript-eslint/naming-convention
-        AREA: {unitSystems: [PresentationUnitSystem.Metric], format: formatProps},
+        AREA: { unitSystems: [PresentationUnitSystem.Metric], format: formatProps },
       };
 
-      const manager: PresentationManager = new PresentationManager({defaultFormats: map});
-      const descriptor = await manager.getContentDescriptor({
-        imodel,
-        rulesetOrId: ruleset,
-        keys,
-        displayType: "Grid",
-        requestContext: new ClientRequestContext(),
-        unitSystem: PresentationUnitSystem.Metric,
-      });
-      expect(descriptor).to.not.be.undefined;
-      const field = findFieldByLabel(descriptor!.fields, "cm2")!;
-      expect(field).not.to.be.undefined;
-      const content = await manager.getContent({ imodel, rulesetOrId: ruleset, keys, descriptor: descriptor!, requestContext: new ClientRequestContext(), unitSystem: PresentationUnitSystem.Metric });
-      const displayValues = content!.contentSet[0].displayValues.rc_generic_PhysicalObject_ncc_MyProp_areaElementAspect as DisplayValuesArray;
-      expect(displayValues.length).is.eq(1);
-      expect(((displayValues[0] as DisplayValuesMap).displayValues as DisplayValuesMap)[field.name]!).to.eq("150.1235 cm²");
+      await checkExpectedAreaDisplayValue("150.1235 cm²", PresentationUnitSystem.Metric, imodel, keys, ruleset, defaultFormats);
     });
   });
 
 });
+
+async function checkExpectedAreaDisplayValue(value: string, unitSystem: PresentationUnitSystem, imodel: IModelDb, keys: KeySet, ruleset: Ruleset, defaultFormats?: {[phenomenon: string]: UnitSystemFormat} ){
```

Comment:
I'd make the following changes:
- Move the function into the scope of `describe("Property value formatting"` - this would allow to skip passing imodel, keys and ruleset as arguments.
- Return the display value instead of having the expectation in the function. Then, make the expectation inside the test function, e.g. `expect(await getAreaDisplayValue(PresentationUnitSystem.Metric, defaultFormats)).to.eq("150.1235 cm²")`